apps/storybook-react-native/.storybook/config.js

```diff
@@ -0,0 +1,12 @@
+import { addDecorator } from '@storybook/react-native';
+import {
+  Theme,
+  ThemeProvider,
+  ColorScheme,
+} from '@metamask/design-system-react-native';
```

Comment:
This makes sense to me as opposed to importing it from `@metamask/design-system-twrnc-preset`. It will also save us time in moving it to a new package so we can get components working